docs/user/filter_expressions.rst

```diff
@@ -222,6 +222,39 @@ filter expressions.
 
    Boolean specifying whether the user has enabled Do Not Track.
 
+.. js:attribute:: normandy.experiments
+
+   Object with several arrays containing the unique slugs for experiments that
+   the user has participated in. Currently, this is limited to preference
+   experiments.
+
+   .. js:attribute:: normandy.experiments.all
+
+      Array of experiment slugs for every experiment that the user has  enrolled
+      in, whether currently active or expired.
+
+   .. js:attribute:: normandy.experiments.active
+
+      Array of experiment slugs for active experiments that the user is enrolled
+      in.
+
+   .. js:attribute:: normandy.experiments.expired
+
+      Array of experiment slugs for expired experiments that the user has
+      enrolled in.
+
+   .. code-block:: javascript
+
+      // Target clients that have ever participated in the "australis"
+      // experiment, including clients that are currently running it
+      "australis" in normandy.experiments.all
```

Comment:
You can't do this in JS, ie:

```js
"foo" in ["foo"]
```

returns false.

Also, because order isn't important here and there should be no duplicates, technically Sets would be better data structures to use here. However, I don't know if jexl can deal with that. If so, that would be the best, IMO. Otherwise, I guess these examples should say:

```js
normandy.experiments.all.includes("australis")
```
etc.

(for sets, substitute "has" for "includes").